docs/content/en/docs/Coding Milestones/PoC2023q1/roadmap-uses.md

```diff
@@ -0,0 +1,101 @@
+---
+categories: ["Coding", "Sprints", "Milestones", "PoC"]
+tags: ["code","milestone","poc2023q1"] 
+title: "Possible Roadmaps for Particular Use Cases"
+linkTitle: "Details"
+weight: 2
+---
+
+{{% pageinfo %}}
+This document outlines thoughts about how soon some particular use cases can work.
+{{% /pageinfo %}}
+
+## Background
+
+The outline [mentions features that need not be implement at
+first](outline/#development-roadmap).  In the following sections we
+consider some particular use cases.
+
+## MVI
+
```

Comment:
Would be good to link docs or description if exists

Reply:
Are you looking for a link to MVI docs - e.g. https://www.ibm.com/products/maximo/visual-inspection ?

Reply:
@ezrasilvera , I think that question is to you.